rpcs3/Emu/System.h

```diff
@@ -88,12 +89,14 @@ class Emulator final
 	std::string m_title_id;
 	std::string m_title;
 	std::string m_app_version;
+	std::string m_hash;
 	std::string m_cat;
 	std::string m_dir;
 	std::string m_sfo_dir;
 	std::string m_game_dir{"PS3_GAME"};
 	std::string m_usr{"00000001"};
 	u32 m_usrid{1};
+	named_thread<SocketIPC> m_ipc_server;
```

Comment:
I'm assuming this can just be placed analog to the other global threads like the progress are thread.

Reply:
No, those are governed by the lifetime of objects inside fxo, which resets on game boot. IPC server needs to be alive from rpcs3 startup to shutdown.